README.md

```diff
@@ -1 +1,30 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
+# Hello World
+
+> This is the first project of Microverse
+
+![screenshot](./home_page.png)
+
+## Built With
+
+- HTML
+- CSS
+
+## Authors
+
+👤 **Mirouhml**
+
+- GitHub: [@mirouhml](https://github.com/mirouhml)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ammar-hamlaoui-514909189/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
```

Comment:
- [ ] Your Readme.md looks perfect!  However, there is a little thing... Please add an MIT license to your project!

You can find it in this [link..](https://github.com/microverseinc/readme-template/blob/master/MIT.md)
> Please remember to personalize your MIT license!